dotcom-rendering/src/components/Dropdown.tsx

```diff
@@ -208,6 +209,14 @@ const dropdownButtonNotificationBadgeStyles = css`
 	margin-top: -3px;
 `;
 
+const dropdownItemContainerStyles = css`
+	display: flex;
+`;
+
+const dropdownItemTextContainerStyles = css`
+	display: block;
+`;
```

Comment:
Would moving the `NotificationBadge` inside the anchor tag not be sufficient?

Reply:
I think I tried that but it didn't have the effect I needed.